app/src/main/java/com/github/se/polyfit/viewmodel/recipe/RecipeRecommendationViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.se.polyfit.viewmodel.recipe
+
+import androidx.lifecycle.ViewModel
+import com.github.se.polyfit.data.api.SpoonacularApiCaller
+import com.github.se.polyfit.model.recipe.Recipe
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+
+@HiltViewModel
+class RecipeRecommendationViewModel
+@Inject
+constructor(private val spoonacularApiCaller: SpoonacularApiCaller) : ViewModel() {
+
+  suspend fun recipeFromIngredients(ingredients: List<String>): List<Recipe> {
+    val recipesResponse =
+        withContext(Dispatchers.IO) { spoonacularApiCaller.recipeByIngredients(ingredients) }
```

Comment:
Shouldn't be the default ? I remember we talked about it, and you told me to let `.DEFAULT` and it will pick the best thread.

Reply:
You're completely right. Thank you for catching that !